converters/fromZigbee.js

```diff
@@ -1505,6 +1505,20 @@ const converters = {
             return {action: `brightness_down_release`};
         },
     },
+    livolo_switch_dev_change: {
+        cid: 'genOnOff',
+        type: 'devChange',
+        convert: (model, msg, publish, options) => {
+            const status = msg.data.data.onOff;
+            const payload = {};
+            payload['state1'] = status & 1 ? 'ON' : 'OFF';
+            payload['state2'] = status & 2 ? 'ON' : 'OFF';
```

Comment:
Can this be changed to `state_left` and `state_right`. In this way we keep it the same as the QBKG03LM

Reply:
Does the device with one button will have ```state_left``` only?

Reply:
It seems that ```modelID``` is identical for all devices. I didn't see any other variants in https://github.com/Koenkk/zigbee2mqtt/issues/592

Reply:
As we cannot distinguish between 1 and 2 gang, I propose to change it to:

```js
payload['state_left'] = status & 1 ? 'ON' : 'OFF';
payload['state_right'] = status & 2 ? 'ON' : 'OFF';
```

The 1 gang will only response to `state_left`.